pyscf/neo/grad.py

```diff
@@ -9,6 +9,137 @@
 from pyscf.grad import rhf as rhf_grad
 from pyscf.lib import logger
 from pyscf.scf.jk import get_jk
+from pyscf.dft.numint import eval_ao, eval_rho, _dot_ao_ao, _scale_ao
+
+
+def eval_xc_nuc(epc, rho_e, rho_n):
```

Comment:
You don't need to copy the function here. Just use the function in neo/ks.py. Use `_, vxc = eval_xc_nuc` to drop exc if you don't need it.